readme.md

```diff
@@ -392,6 +392,8 @@ This list is ordered based on the name without prefix, so excluding
     — Ensure that the `href` of links has no trailing slash
 *   [`remark-lint-write-good`](https://github.com/zerok/remark-lint-write-good)
     — Wrapper for `write-good`
+*   [`remark-lint-double-link`](https://github.com/Scrum/remark-lint-double-link)
+    — This rule ensures that there will be no duplicate links URL in the list.
```

Comment:
```suggestion
    — Ensure the same URL is not linked multiple times
```

How about this? The other external rules start with `Ensure` and end without a `.`. I’d prefer them to be similar in that sense

Thanks!

Reply:
Thanks, it looks better.  I will make changes.

Reply:
(you can click on the button here on GH!)

Reply:
Yes, I wrote from a mobile application and there is no file editing there or I didn’t find it (